lesson-02-assignment/lesson-02-assignment.rb

```diff
@@ -0,0 +1,63 @@
+#1
+
+Course.create(coding_class_id: "1", trimester_id: "8", max_enrollment: "25")
+Course.create(coding_class_id: "2", trimester_id: "8", max_enrollment: "25")
+Course.create(coding_class_id: "3", trimester_id: "8", max_enrollment: "25")
+Course.create(coding_class_id: "4", trimester_id: "8", max_enrollment: "25")
+Course.create(coding_class_id: "5", trimester_id: "8", max_enrollment: "25")
+
+#2
+
+Student.create(first_name: "Val", last_name: "Lyzhyn", email: "val@test.com")
+student_id = Student.last.id
+Enrollment.create(course_id: "1", student_id: student_id)

```

Comment:
We wanted to enroll a student for the Intro class of the Spring 2026 trimester specifically. How could we get the info for that?